### DISCUSSION TITLE
3.7.0 Use System.Text.Json

### DISCUSSION THREAD

**Discussion by newloopadmin** [quartznet/quartznet#2116] [Q&A] [answered]
I have a solution where my System.Text.Json serializer is configured with a lot of different converters.
Can I get Quartz (Quartz.AspNetCore 3.7.0) to use that serializer?

Looking in the code, I found:
https://github.com/quartznet/quartznet/tree/main/src/Quartz.Serialization.SystemTextJson

So it seems there is some sort of implementation of System.Text.Json, but how do I use it? I could use the UseSerializer<> method, but then I would miss the converters seen here:
https://github.com/quartznet/quartznet/blob/main/src/Quartz.Serialization.SystemTextJson/JsonConfigurationExtensions.cs

Installing Quartz.Serialization.Json 3.7.0 uses newtonsoft.

**Comment by lahma** [comment, marked as answer]
3.x only supports Newtonsoft.JSON. `main` branch reflects 4.x development and there's no database persistence support there yet IIRC. Newtonsoft.JSON is quite battle-tested sho it should be a fine choice.

**Reply by newloopadmin** [reply]
Ok thanks
